src/rebuild.rs

```diff
@@ -254,3 +254,21 @@ pub fn after_template_change(site: &mut Site, path: &Path) -> Result<()> {
         },
     }
 }
+
+/// What happens when the config is changed
+pub fn after_config_change(path: &Path, interface: &str, port: &str, config_file: &str) -> Result<Site> {
```

Comment:
this looks suspiciously like the the beginning of the `serve` function, can this just be extracted from it and stay in `serve.rs`?